config/Config.py

```diff
@@ -1,37 +1,15 @@
 import os
 
-common_caps = {
-    "browserstack.user" : "BROWSERSTACK_USERNAME",
-    "browserstack.key" : "BROWSERSTACK_ACCESS_KEY",
-    "build" : "browserstack-build-1",
-    "browserstack.debug" : "true"
-}
-
 envs = [{
-    "os" : "Windows",
-    "os_version" : "10",
-    "browser" : "Chrome",
-    "browser_version" : "latest"
+    "browser": "Chrome",
 },
-{
-    "os" : "OS X",
-    "os_version" : "Big Sur",
-    "browser" : "Safari",
-    "browser_version" : "latest"
+    {
+    "browser": "Safari",
 },
-{
-    "device" : "Samsung Galaxy S22",
-    "os_version" : "12"
+    {
+    "browser": "Firefox",
```

Comment:
Does user has to define num of platforms in yml = entries in this env?

Reply:
This is just a reference of what the user might already have before sdk.. this will be ignored and yml honored..

Reply:
What if we remove all the envs?

Reply:
will be overridden so can keep this..